modules/EnsEMBL/Web/ZMenu/Gene.pm

```diff
@@ -34,17 +34,19 @@ sub _content {
   if ($species_defs->GENOMIC_UNIT eq 'bacteria') {
     $zmenu_label = $xref[0] ? $xref[0]." (".$object->stable_id.")" : $object->stable_id;
   }
-  
-  $self->add_entry({
-    type  => 'Gene',
-    label => $zmenu_label,
-    link  => $hub->url({
-      type => 'Gene',
-      action => 'Summary',
-      g => $g,
-#     __clear => '1' #gets rid of the url variation data in order to avoid a bug                                      
-    })
-  });
+
+  if($zmenu_label) {
+    # $self->add_entry({
```

Comment:
why do you have these line of code commented? If its not being used can you please remove it